test/unit/spec/nbextensions/bulkImportCell/bulkImportCell-spec.js

```diff
@@ -394,13 +400,39 @@ define([
             });
         });
 
-        describe('cancel', () => {
-            ['launching', 'inProgress', 'inProgressResultsAvailable'].forEach((testCase) => {
-                it(`should cancel the ${testCase} state and return to a previous state`, () => {
+        describe('cancel and reset', () => {
+            const tests = [
+                {
+                    state: 'inProgressResultsAvailable',
+                    action: 'cancel',
+                },
+                {
+                    state: 'inProgress',
+                    action: 'cancel',
+                },
+                {
+                    state: 'launching',
+                    action: 'cancel',
+                },
+                {
+                    state: 'jobsFinished',
+                    action: 'reset',
+                },
+                {
+                    state: 'jobsFinishedResultsAvailable',
+                    action: 'reset',
+                },
+                {
+                    state: 'error',
+                    action: 'reset',
+                },
+            ];
+            tests.forEach((testCase) => {
+                it(`should ${testCase.action} the ${testCase.state} state and return to a previous state`, () => {
```

Comment:
since 'cancel' and 'reset' do the same things but with different dialogs, we can share the same test framework